features/forum_categories.feature

```diff
@@ -0,0 +1,23 @@
+Feature: Basic Florum Categories
+  "As a user,
+  In order to sort through various plant topics,
+  I would like to see a main forum page with separate categories"
+
+  Background:
+    Given the following user exists
+    | name    |
+    | Fat Bob |
+    | Slim Jim|
+    And the following post exists
+    | title        | text          | user     | category     |
+    | Codeacademy  | Smooth-ride   | Fat Bob  | general      |
+    | Hacklife     | smaaaahht     | Slim Jim | marketplace  |
+    And I am logged in as "Fat Bob" 
+
+  Scenario: Visitor can see basic forum categories
+    When I visit the site
+    When I click "FLORUM"
+    Then I should see "MARKETPLACE"
+    Then I should see "GENERAL"
+    Then I should see "GROWTH & CARE"
```

Comment:
Should we be able to see this category when there are no posts with that category?

Reply:
Why not?  We still need to know what the categories of the forum are so we know where we should post